src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectProtocolValidator.cs

```diff
@@ -93,7 +99,7 @@ public OpenIdConnectProtocolValidator()
             RequireAzp = false;
             RequireNonce = true;
             RequireState = true;
-            RequireSub = false;
+            RequireSub = RequireSubByDefault;
```

Comment:
I don't believe this is needed since you are setting it at the property with } = RequireSubByDefault